lib/resty/limit/count.lua

```diff
@@ -80,6 +111,14 @@ function _M.incoming(self, key, commit)
     return 0, remaining
 end
 
+function _M.incoming(self, key, commit)
```

Comment:
This way should be better:
```
local function incoming_new
local function incoming_old
_M.incomming = incr_support_init_ttl and incoming_new or incoming_old
```

Reply:
@doujiang24  fixed, please review again, thanks